packages/sit-onyx/src/components/OnyxAppLayout/OnyxAppLayout.vue

```diff
@@ -0,0 +1,84 @@
+<script lang="ts" setup>
+import { useSlots } from "vue";
+
+const props = withDefaults(
+  defineProps<{
+    /** Whether the nav bar will stick to the left or to the top of the app */
+    navBarAlignment: "top" | "left";
```

Comment:
```suggestion
    navBarAlignment?: "top" | "left";
```

If it has a default value, it should be optional